core/tests/protractor_desktop/topicsAndSkillsDashboard.js

```diff
@@ -54,12 +51,47 @@ describe('Topics and skills dashboard functionality', function() {
     await topicsAndSkillsDashboardPage.get();
   });
 
-  it('should add a new topic to list', async function() {
+  it('should add a new topic to list and delete it', async function() {
     await topicsAndSkillsDashboardPage.expectNumberOfTopicsToBe(0);
-    await topicsAndSkillsDashboardPage.createTopic('Topic 1', true);
+    await topicsAndSkillsDashboardPage.createTopic(
+      'Topic 1', 'Topic 1 description', 'Mathematics', true);
```

Comment:
Can we make this more reliably unique? Maybe `topic1TopicsAndSkillsDashboard`?

Reply:
Done.